packages/server/events-processing-platform/.env

```diff
@@ -1,6 +1,17 @@
 SERVICE_NAME: events-processing-platform
 
-EVENT_STORE_CONNECTION_STRING: esdb://localhost:2113?tls=false
+EVENT_STORE_CONNECTION_STRING: esdb://localhost:2113
+EVENT_STORE_CONNECTION_TLS_DISABLED: true
+EVENT_STORE_CONNECTION_TLS_VERIFY_CERT: false
+EVENT_STORE_CONNECTION_KEEP_ALIVE_TIMEOUT: 10000
+EVENT_STORE_CONNECTION_KEEP_ALIVE_INTERVAL: 10000
+EVENT_STORE_CONNECTION_USERNAME:
```

Comment:
These new env vars need to also be added in 
deployment/infra/k8s/customer-os-api.yaml

Reply:
these are events-platform vars.
whole app to be added.
will create a task

Reply:
the app has already been added by silviu